mitielib/include/mitie/word_morphology_feature_extractor.h

```diff
@@ -93,6 +94,7 @@ namespace mitie
                   iterator range.
         !*/
         {
+            std::vector<dlib::uint16> hits;
```

Comment:
Why is this here?  

This is the third time I've seen variables like this added that just alias existing variables.  Please don't do this, unless there is some very good reason to do it (which is not apparent to me).